Source/ParserProperty.swift

```diff
@@ -118,11 +118,11 @@ public class ParserPropertyMaker {
         - returns: The created parser property.
     */
     @discardableResult
-    public func propertyForKeyPath(
-        _ keyPath: String,
+    public func property(
+        forKeyPath keyPath: String,
         type: ParserPropertyType,
         optional: Bool = false,
-        decodedToType decodableType: Decodable.Type?)
+        decodableType: Decodable.Type?)
```

Comment:
It seemed logical here to replace `decodedToType` with simply `decodableType`. At this point in the method signature, we've lost grammatical correctness. Whether we use `propertyForKeyPath` or `property(forKeyPath:)` is really the tough decision. Apple has contradictory info about this case in the API guidelines.